src/test/java/seedu/algobase/logic/commands/DefaultModelStub.java

```diff
@@ -84,11 +83,36 @@ public void setProblem(Problem target, Problem editedProblem) {
         throw new AssertionError("This method should not be called.");
     }
 
+    @Override
+    public void updateFilteredPlanList(Predicate<Plan> predicate) {
+        throw new AssertionError("This method should not be called.");
+    }
+
     @Override
     public ObservableList<Problem> getFilteredProblemList() {
         throw new AssertionError("This method should not be called.");
     }
 
+    @Override
+    public boolean hasPlan(Plan plan) {
+        return false;
```

Comment:
Hmm should be `throw new AssertionError("This method should not be called.");` right?